letter-sender/src/main/java/com/christmas/letter/sender/model/Letter.java

```diff
@@ -0,0 +1,17 @@
+package com.christmas.letter.sender.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class Letter {
+    String email;
```

Comment:
I would expect to see some validations here, the email should be required, it should match text@text.domain. You could use @Data instead of @Getter, @Setter etc, and the properties should be private.

Reply:
Done